src/index.html

```diff
@@ -6,9 +6,351 @@
+              <h3>L.A. University of Applied Science</h3>
+              <h4>2002 - 2006</h4>
+              <p>
+                This is a description of your education. Concisely describe your
+                certifiate, degree or course in a way that’s easy for readers to
+                scan quickly.
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <!-- Hobbies & Interests Slider-->
+      <div class="slider fade">
+        <div class="top">
+          <div class="title">
+            <h2>HOBBIES & INTERESTS</h2>
+          </div>
+          <div class="media">
+            <a href="">DOWNLOAD MY FULL RESUME</a>
+          </div>
+        </div>
+        <div class="bottom">
+          <div class="bottom-content">
+            <div class="hobbies">
+              <i class="fas fa-ad"></i>
+              <p>Cinema</p>
+            </div>
+            <div class="hobbies">
+              <i class="fas fa-volleyball-ball"></i>
+              <p>Sports</p>
+            </div>
+            <div class="hobbies">
+              <i class="fas fa-theater-masks"></i>
+              <p>Thratre</p>
+            </div>
+            <div class="hobbies">
+              <i class="fas fa-utensils"></i>
+              <p>BBQ</p>
+            </div>
+            <div class="hobbies">
+              <i class="fas fa-glass-cheers"></i>
+              <p>Party</p>
+            </div>
+            <div class="hobbies">
+              <i class="fas fa-ad"></i>
+              <p>Cinema</p>
+            </div>
+            <div class="hobbies">
+              <i class="fas fa-cookie"></i>
+              <p>Cooking</p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="dot-sections">
+        <span class="dot" onclick="currentSlide(1)"></span>
+        <span class="dot" onclick="currentSlide(2)"></span>
+        <span class="dot" onclick="currentSlide(3)"></span>
+        <span class="dot" onclick="currentSlide(4)"></span>
+      </div>
+    </div>
+
+    <!-- Skill Section-->
+    <div class="skills-section">
+      <h2>SKILLS</h2>
+      <div class="skill">
+        <p class="skill_name">Microsoft Excel</p>
+        <p class="skill_level">Advanced</p>
+      </div>
+      <div class="skill">
+        <p class="skill_name">Microsoft Word</p>
+        <p class="skill_level">Advanced</p>
+      </div>
+      <div class="skill">
+        <p class="skill_name">Adobe</p>
+        <p class="skill_level">Proficient</p>
+      </div>
+      <div class="skill">
+        <p class="skill_name">PowerPoint</p>
+        <p class="skill_level">Proficient</p>
+      </div>
+      <div class="skill">
+        <p class="skill_name">French</p>
+        <p class="skill_level">Native</p>
+      </div>
+      <div class="skill">
+        <p class="skill_name">Spanish</p>
+        <p class="skill_level">Advanced</p>
+      </div>
+    </div>
+
+    <!-- Contacts Section-->
```

Comment:
Hi Frankie,
I can see you are confident with you html/css skills. Your pull request scope way beyond what we required.
However, styling wise we are not looking at all the other sections besides "QUICK ID" and "CONTACT ME" section.

Please take a close look at our homework requirement:

> "Follow the design to style the two sections with the basic CSS styles that we learnt, such as the font, background, color, border and width, height, line-height and etc."

It makes our review very hard to focus on the parts we wanted to see. We would be really happy if you can just stick to what we want you to do next time.

Apart from that, look at the design closely. I ran you code on my local, you build is still quite different from design. A good frontend dev should really care about the details and try to build according to the design.